checkov/terraform/checks/graph_checks/azure/AzureConfigMSSQLwithAD.yaml

```diff
@@ -0,0 +1,21 @@
+metadata:
+  id: CKV2_AZURE_27
```

Comment:
I think you accidentally committed new checks to this PR 😄

Reply:
Uh oh. Sorry, will be updating it now.

Reply:
Done. Kindly check now